lib/MetaCPAN/Web/Controller/Release.pm

```diff
@@ -162,54 +134,52 @@ sub view : Private {
 }
 
 sub _files_to_categories {
-    my ( $self, $release, $files ) = @_;
+    my $self  = shift;
+    my %files = map +( $_->{path} => $_ ), @_;
+    my @files = sort keys %files;
+
     my $ret = +{
-        provides          => [],
-        documentation     => [],
-        documentation_raw => [],
-        modules           => [],
+        provides      => [],
+        documentation => [],
+        modules       => [],
+        other         => [],
+        examples      => [],
     };
 
-    my %skip;
-
-    for my $f (@$files) {
-        next if $f->{documentation};
-        my @modules
-            = is_arrayref( $f->{module} ) ? @{ $f->{module} } : $f->{module};
-        for my $module ( grep {defined} @modules ) {
+    for my $f ( @files{@files} ) {
```

Comment:
isn't this just `values %files`? i don't see any code changing the hash or the array.